app/controllers/fast_controller.rb

```diff
@@ -78,49 +63,25 @@ def num_records
     Settings.autocomplete_lookup.num_records
   end
 
-  # Split into left-anchored matches to query and other matches.
-  def partition_suggestions(suggestions, query)
-    Array(suggestions).partition { |suggestion| suggestion.keys.first.downcase.start_with?(query.downcase) }
-  end
-
-  def merge_suggestions(suggestions1, suggestions2, query)
-    left_match_suggestions1, other_suggestions1 = partition_suggestions(suggestions1, query)
-    left_match_suggestions2, other_suggestions2 = partition_suggestions(suggestions2, query)
-
-    # Order is left match (non-wildcard) alpha, left match (wildcard) alpha, other matches alpha
-    left_match_suggestions = (sort_suggestions(left_match_suggestions1) + sort_suggestions(left_match_suggestions2))
-                             .take(num_records)
-    num_other_suggestions = num_records - left_match_suggestions.size
-    other_suggestions = (other_suggestions1 + other_suggestions2)
-                        .uniq
-                        .take(num_other_suggestions)
-
-    left_match_suggestions + sort_suggestions(other_suggestions)
-  end
-
-  def sort_suggestions(suggestions)
-    suggestions.sort { |suggestion1, suggestion2| suggestion1.keys.first <=> suggestion2.keys.first }
-  end
-
   URI_PREFIX = 'http://id.worldcat.org/fast/'
 
   def key(idroot, type)
     "#{URI_PREFIX}#{idroot.delete_prefix('fst').to_i}/::#{cocina_type(type)}"
   end
 
   # Map of FAST types to Cocina types.
-  SUBJECT_TYPES = {
-    'person' => 'person',
-    'corporate' => 'organization',
-    'event' => 'event',
-    'geographic' => 'place',
-    'title' => 'title',
-    'form' => 'genre',
-    'meeting' => 'conference',
-    'chronological' => 'time'
+  TAG_TYPES = {
+    100 => 'person',
+    110 => 'organization',
+    111 => 'event',
+    151 => 'place',
+    130 => 'title',
+    155 => 'genre',
+    148 => 'time',
+    # NOTE: Missing: 'meeting' => 'conference'
   }.freeze
```

Comment:
@amyehodge @arcadiafalcone @edsu The results from the new API do not include a type, but do include a tag. OCLC provide some sample JavaScript code that suggest a MARC tag mapping like the above. Note that I could not find any evidence that events and meetings are separate in this dataset.